examples/strands/strands.bal

```diff
@@ -8,6 +8,9 @@ function userSpeakerService(string userName) {
 }
 
 public function main() {
+
+    io:println("In function worker");
```

Comment:
Don't you have to change the output?

Reply:
The output validation has been disabled for this BBE since the order of logs may differ.

Reply:
Meant the out file, but looks like it's already there.